nntrainer/layers/bn_layer.cpp

```diff
@@ -63,15 +63,15 @@ int BatchNormalizationLayer::initialize(Manager &manager) {
   if (weights.empty()) {
     weights.reserve(4);
     weights.emplace_back(dim, initializers[BNParams::mu],
-                         WeightRegularizer::NONE, 1.0f, false,
+                         WeightRegularizer::NONE, 1.0f, false, true,
```

Comment:
Nice catch! how about making weight ctor explicit to prevent further mistake?
```diff
- Weight(
+ explicit Weight(f
    const TensorDim &dim,
    const WeightInitializer init = WeightInitializer::WEIGHT_XAVIER_UNIFORM,
    const WeightRegularizer reg = WeightRegularizer::NONE,
    const float reg_const = 1.0f, bool train = true, bool alloc_now = true,
    std::string name = "");
```

Reply:
added for weight and var_grad.